students/jemielity_kamil/lesson_04_unit_testing/the_collatz_sequence_tests.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+from students.jemielity_kamil.lesson_03_functions.the_collatz_sequence import collatz
+
+
+class MyTest(unittest.TestCase):
+
+    def test_collatz_string_value(self):
+        self.assertRaises(TypeError, collatz, 'aeou')
+
+    def test_collatz_of_number_8(self):
+        self.assertEqual(collatz(8), 4, 'Colltaz of number 8 != 4')
```

Comment:
Just note that description 'Colltaz of number 8 != 4' is not necessary as the one raised by pytest is self explaining
```
self.assertEqual(collatz(8), 4)
AssertionError: 8 != 4
```